### PR TITLE
docs: move metrika out

### PR DESCRIPTION
**Pull request by zxqfox**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,7 +88,7 @@ Declare a counter in your project BEMJSON:
 Set the following parameters of the block:
 
 * `id` — required
-* `webvisor` — optoinal
+* `webvisor` — optional
```
Comment (remnev) [inline README.md]: What was changed here? :)

File: README.md [hunk 2]
```diff
@@ -98,7 +98,7 @@ Set the following parameters of the block:
  {
     block: 'yandex-metrica',
     params: {
-        id: configs.metrika.id, // required param
+        id: '0000000', // required param
```
Comment (remnev) [inline README.md]: May be note as Number?

File: README.md (hunk 1 above)
Comment (zxqfox) [inline README.md]: just a typo in `optoinal`

File: README.md (hunk 2 above)
Comment (zxqfox) [inline README.md]: Don't think so because there are no Integers in JS.

File: README.md (hunk 1 above)
Comment (remnev) [inline README.md]: Oh, exactly!

File: README.md (hunk 2 above)
Comment (remnev) [inline README.md]: What is the relationship between Number and Integers I don't figure out. But that to be the same as type of other blocks, yes String is ok.

**Comment by remnev** [comment]
:+1:

**Pull request closed without merging**